sql/pgsql/src/TPgSQLStatement.cxx

```diff
@@ -850,25 +870,34 @@ Bool_t TPgSQLStatement::SetTime(Int_t npar, Int_t hour, Int_t min, Int_t sec)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Set parameter value as date & time.
+/// Set parameter value as date & time, in UTC.
```

Comment:
Are the mysql etc implementations also expecting the time to be passed in UTC? Else we have an asymmetry between the different implementations that's surprising for users. (I understand the asymmetry between get/set of pgsql, but I'm trying to understand the general interface intent here...)

Reply:
Good morning. Every implementation of the TSQLStatement class can use own approach to save and load timestamps if it works correct. As TPgSqlStatement returns always UTC time, so we should save it also in UTC otherwise we have the different time values as shown above. I think to store UTC time for TDatime is the right solution because TDatime "has no support for time zones"., but it can be read from different geographical locations. So, in UTC internal case a user puts and gets the same time value. If it works as should it will not be a surprise for users, which deals with ROOT classes  - he may not know internal representation.

Reply:
Thanks, sounds convincing! @linev @pcanal opinions? (Both are on vacation so this might take a bit...)